### ISSUE TITLE
Virtual Scroll creates extra object when a pipe is used within the list

### ISSUE THREAD

**Issue by pmurpmur** [driftyco/ionic#6349]
#### Short description of the problem:
Whenever I use a pipe within a virtual scroll list it creates an extra item  in the list (which is seen as [object Object] when logged to the console). This is a problem because it displays an extra item and if the list is a list of objects the application throws an error saying I am trying to read a property of undefined and then the app crashes. I have seen this with Angular2's date pipe, Angular2's uppercase pipe and my own custom pipes.

#### What behavior are you expecting?
I want to be able to use pipes within virtual scroll.

**Steps to reproduce:**
1. Setup virtual scroll
2. Add pipe to an item within the virtual scroll (not the virtual scroll itself)

```
<ion-list [virtualScroll]="items">
    <ion-item  *virtualItem="#item">
        <div>{{item | uppercase}}</div>
    </ion-item>
</ion-list>
```

This is seen using Ionic Version 2.6

#### Plunker that shows an example of the issue
http://plnkr.co/edit/N2znBvohHpljj5qyZb3Y?p=preview

#### Ionic Info
Cordova CLI: 6.0.0
Gulp version:  CLI version 3.9.0
Gulp local:   Local version 3.9.1
Ionic Framework Version: 2.0.0-beta.6
Ionic CLI Version: 2.0.0-beta.25
Ionic App Lib Version: 2.0.0-beta.15
ios-deploy version: 1.8.6 
ios-sim version: 5.0.6 
OS: Mac OS X El Capitan
Node Version: v5.10.1
Xcode version: Xcode 7.3 Build version 7D175

**Comment by newnottakenname** [comment]
This seems to be fixed in the latest beta (2.0.0-beta.10)
http://embed.plnkr.co/gU9hcHgsS13HHgLtQ6Ic/

**Comment by jgw96** [comment]
Thanks @newnottakenname! I can confirm that it seems to be fixed in the latest beta.
